core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -284,5 +284,7 @@ abstract class SparkFunSuite
     def setThreshold(threshold: Level): Unit = {
       _threshold = threshold
     }
+
+    def nonNullLogEvents(): Seq[LogEvent] = loggingEvents.filterNot(_ == null)
```

Comment:
Actually,  I tried overwriting `append(loggingEvent: LogEvent)` to filter out the null events, but it doesn't work. I don't know why there are null events yet but I think we should fix this to unblock the CI tests.